### PR TITLE
Feature: init

### PR DESCRIPTION
**Pull request by DanielSiepmann**

### CODE REVIEW

File: Documentation/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+html_theme = 'alabaster'
+
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+html_theme_options = {
+    'description': 'TYPO3 Extension to integrate search services.',
```
Comment (justusmoroni) [inline Documentation/source/conf.py]: This is the default description?

Same for the twitter extension: https://search-core-twitter.readthedocs.io/en/latest/

File: Documentation/source/configuration.rst [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+----------
+
+All information are fetched from Facebook via REST API. It's possible to define all parameters to
+use via TypoScript. Currently only ``fields`` is known to work:
+
+.. code-block:: typoscript
+
+   plugin.tx_searchcore.settings.indexing {
+       facebookPosts {
+           parameters {
```
Comment (justusmoroni) [inline Documentation/source/configuration.rst:117]: `parameters` is in `parameters`. Is this correct?

File: Documentation/source/installation.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Setup
+-----
+
+To be able to index anything from Facebook, you need to create an app to get the necessary
+credentials. New and existing apps can be managed at https://developers.facebook.com/apps/ .
+
+Afterwards you need to enable the extension through the extension manager and need to configure the
+indexing through TypoScript.
+
+.. _download: https://github.com/codappix/search_core_facebook/archive/develop.zip
```
Comment (justusmoroni) [inline Documentation/source/installation.rst]: We shouldn't link to the development state of the extension.

File: ext_emconf.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    'description' => 'Search core implementation for Facebook.',
+    'category' => 'be',
+    'clearCacheOnLoad' => 1,
+    'constraints' => [
+        'depends' => [
+            'typo3' => '7.6.0-8.7.99',
+            'php' => '7.0.0-7.2.99',
+            'search_core' => '',
+        ],
+        'conflicts' => [],
```
Comment (justusmoroni) [inline ext_emconf.php:14]: Remove if we don't have conflicts?

**Review by justusmoroni — CHANGES_REQUESTED** [review]

File: ext_emconf.php (hunk 4 above)
Comment (DanielSiepmann) [reply ext_emconf.php:14]: I'll keep it, I'm even not sure whether it's required.

**Review by justusmoroni — APPROVED** [review]

**Pull request merged**